generic-ec/src/lib.rs

```diff
@@ -212,6 +213,7 @@ pub mod traits {
     }
 }
 
+#[cfg(feature = "serde")]
 pub mod serde;
```

Comment:
Why this change is required?

Reply:
If you did not enable serde, it shows unused imports in this module.

Reply:
That's strange, but that only means that we need to fix unused imports. `serde` module needs to be present even if feature is disabled

Reply:
Done.